components/stream-provider/README.md

```diff
@@ -0,0 +1,94 @@
++ `HEIGHT` (number) - the window height
++ `WIDTH` (number) - the window width
++ `EXP_HIDE_STREAMER` (boolean) - experiment flag for hiding the streamer window
++ `TWILIO_ACCOUNT_SID` (string) - a Twilio AccountSid required to get a Network Traversal Service Token
++ `TWILIO_AUTH_TOKEN` (string) - a Twilio AuthToken required to get a Network Traversal Service Token
+
+## Turn server
+
+Our service supports both [coturn](https://github.com/coturn/coturn) and [Twilio's STUN/TURN service](https://www.twilio.com/docs/stun-turn).
+In the dotenv file, if `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN` values are set, our service will attempt to get a turn server from Twilio. Otherwise, you can leave them empty to use a stun server or coturn turn server.
+
+## Electron security
+
+Our service follows [electron security guideline](https://electronjs.org/docs/tutorial/security) and enables the following behaviors:
+
++ Enabling [contextIsolation](https://electronjs.org/docs/tutorial/security#3-enable-context-isolation-for-remote-content), which allows scripts running in the renderer to make changes to its javascript environment without worrying about conflicting with the scripts in the electron API or the preload script.
++ Blocking [mouse middle-clicking](https://www.blackhat.com/docs/us-17/thursday/us-17-Carettoni-Electronegativity-A-Study-Of-Electron-Security-wp.pdf), which opens a new window and makes our remote input stop working.
++ Disabling popup dialog for file downloading (when clicking on a link) so that it doesn't interfere with the streamer window.
++ Displaying warning about using [insecure http protocol](https://electronjs.org/docs/tutorial/security#1-only-load-secure-content) or when the streamer window [navigates to a new origin](https://electronjs.org/docs/tutorial/security#12-disable-or-limit-navigation), which is different from the `SERVICE_URL`.
+
+## Running
+
+How to get stream-provider up and running. ⚙
+
+### Docker
+> Note: Your `.env` file should be in the stream-provider package directory (next to `package.json`) - for more details see the
+[Configuration](#configuration) section above.
+> Note: Our [`Dockerfile`](./Dockerfile) has a defined display resolution of `1920x1080` - You may change this as needed.
+
+You'll need [docker](https://docs.docker.com/install/) to build and run. Once you have it, you can build and run:
+
+```
+# build the container (and source)
+docker build . -f components/stream-provider -t browserd-provider:local
+
+# run
+docker run -it --env-file ./components/stream-provider/.env browserd-provider:local
+```
+
+### Locally
+
+> Note: Your `.env` file should be in the project directory (next to `package.json`) - for more details see the
+[Configuration](#configuration) section above.
+
+You'll need [`Node LTS (v10.x.x)`](https://nodejs.org/en/) and `npm` (bundled with the node installer`) to build and run. Once you have
+those, you can install dependencies and run:
+
+```
+# install lerna
```

Comment:
nit: again, hoist to main?

Reply:
So i wrote a brief introduction in the root README and prompted people to install lerna. Is that in line with what you were thinking?